src/cbmc/bmc.h

```diff
@@ -193,6 +193,10 @@ class bmct:public safety_checkert
 
   friend class bmc_all_propertiest;
 
+  // \todo remove these two declarations once test-gen dependency is fixed
+  bool cover(const goto_functionst &goto_functions);
```

Comment:
:question: How does adding a declaration without definition help? Because TG provides a implementation.

Reply:
But ... this isn't declared `virtual`?

Reply:
No - TG compiles and links a copy of `bmc_cover.cpp` with the `bmct::cover` directly.

Reply:
I see - so it's technical debt coming to haunt you 👻